homework-submission/my-project/src/App.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+import './App.css';
+import Post from './Post';
+import Data from './Data';
+
+class App extends Component {
+  render() {
+    const newData = Data.map(item => <Post key={item.id} description={item.description} deadlineDate={item.deadlineDate} />)
```

Comment:
it is a good improve in your code here, but we don't need to change our data just for this idea because the map method have got an internal key that we can use it so you can save your time by doing the map method  as I mentioned before like this:
Data.map( ( item ,  key ) => < TodoItem description={item.description} deadlineDate={item.deadlineDate}  key={key} />)

Reply:
II appreciate your suggestion, but I tried to find another solution and I came out with adding id. I could also use any other unique key, right ? like that:
const newData = Data.map(item => <Post key={item.description} description={item.description} deadlineDate={item.deadlineDate} />)

Reply:
what is the reason that you like it.
you change your data just for your functions but your functions have got this unique key so just use it.
in this case your card has a low cost.